### DISCUSSION TITLE
Colab session terminated, how to know if fine tuning finished?

### DISCUSSION THREAD

**Discussion by gibranpulga** [TheLastBen/fast-stable-diffusion#1928] [Q&A] [answered]
Hey everyone. 
A question about the fine tuning process.
I've started fine tuning a model on pictures of me on the Colab notebook, left the laptop a bit, laptop went to sleep, came back, saw that the Colab session was terminated, and the Training - Start Dreambooth section was stuck in 64%.
I reconnected the the Colab session. The ckpt file was created inside my Google Drive. 
How do I know if the fine tuning completed to 100%? Is the ckpt file only created when it reaches to 100% or it's created by steps?

Do you know ?

**Comment by TheLastBen** [comment, marked as answer]
the ckpt is created only if the training is complete, run the colab, load the session, the go directly to the testing cell
